modules/TestUtils.js

```diff
@@ -53,27 +79,29 @@ export const arrayContains = (array, value, compareValues) => {
 }
 
 /**
- * Returns true if the given string contains the value, false otherwise.
- */
-export const stringContains = (string, value) =>
-  string.indexOf(value) !== -1
-
-/**
- * Returns true if the given object is an array.
+ * Returns true if the given object contains the value, false
+ * otherwise. The compareValues function must return false to
+ * indicate a non-match.
  */
-export const isArray = (object) =>
-  Array.isArray(object)
+export const objectContains = (object, value, compareValues) => {
+  if (compareValues == null)
+    compareValues = isEqual
 
-/**
- * Returns true if the given object is an instanceof value
- * or its typeof is the given value.
- */
-export const isA = (object, value) => {
-  if (isFunction(value))
-    return object instanceof value
+  return Object.keys(value).every(k => {
+    let equals
 
-  if (value === 'array')
-    return Array.isArray(object)
+    if (isObject(object[k])) {
+      equals = objectContains(object[k], value[k], compareValues)
```

Comment:
this might be more concise as an early return, and a top-level return on line 96